math/wxMaxima/files/patch-devel-src_Dirstructure.cpp.diff

```diff
@@ -0,0 +1,15 @@
+--- src/Dirstructure.cpp.orig	2018-06-01 18:24:00.000000000 +0900
++++ src/Dirstructure.cpp	2018-06-03 23:04:26.000000000 +0900
+@@ -296,10 +296,8 @@
+     command = wxT("/Applications/Maxima.app");
+   if (wxFileExists("/Applications/maxima.app"))
+     command = wxT("/Applications/maxima.app");
+-  else if (wxFileExists("/usr/local/bin/maxima"))
+-    command = wxT("/usr/local/bin/maxima");
+-  else if (wxFileExists("/usr/bin/maxima"))
+-    command = wxT("/usr/bin/maxima");
++  else if (wxFileExists("/OPT/LOCAL/bin/maxima"))
++    command = wxT("/OPT/LOCAL/bin/maxima");
```

Comment:
The capitalization here is somewhat unusual. Also, the `reinplace` Portfile command might be a reasonable substitute for the patchfile, someone with more experience might want to say.

Reply:
I mean that "/OPT/LOCAL" is a mark of replacement.
After patching, the next command is executed.

    post-patch {
        reinplace -W ${worksrcpath} "s|/OPT/LOCAL|${prefix}|g" \
                                    src/Dirstructure.cpp
    }

Reply:
Ah, I see.